### PR TITLE
Feature and Refactor

### PR DESCRIPTION
**Pull request by Quannichan**
Design header layouts (menu, header'scomponents)
Edit packaage.json, package-lock.json and .gitignore .

### CODE REVIEW

File: src/stylesheet/_backgrounds.scss [hunk 1]
```diff
@@ -1 +1 @@
-
+$back_ground_header : url("../images/background_header.png");
```
Comment (Lanng21) [inline src/stylesheet/_backgrounds.scss:1]: end of line

File: src/layouts/header.html [hunk 2]
```diff
@@ -1 +1,7 @@
-
+<div class="wrap_header">
+    <div class="image_header">
+    </div>
+    <include src="./components/menu.html" ></include>
+    
+    <include src="./components/header_compo.html" ></include>
+</div>
```
Comment (Lanng21) [inline src/layouts/header.html:7]: pls check git convention : break line, end of line....

File: src/components/header_compo.html [hunk 3]
```diff
@@ -1 +1,15 @@
+<div class="header_component">
+    <div class="wrap_components">
+        <div class="compo1">
+            <p class="head_header">See The Unmatched Beauty Of The Great Lakes</p>
+        </div>
+        <div class="wrap_video_header">
+            <iframe class="video_header" src="https://www.youtube.com/embed/tgbNymZ7vqY">
+            </iframe>
+        </div>
+        <div class="wrap_arrow">
```
Comment (Lanng21) [inline src/components/header_compo.html:10]: use figure tag instead div here

File: src/components/header_compo.html [hunk 4]
```diff
@@ -1 +1,15 @@
+<div class="header_component">
```
Comment (Lanng21) [inline src/components/header_compo.html:1]: Can u use html semantic?

File: src/components/menu.html [hunk 5]
```diff
@@ -1 +1,23 @@
+<div class="wrap_menu">
 
+    <div class="wrap_trademark">
+            <p class="trademark">HOFMANN</p>
+    </div>
+
+<div class="wrap_list1_menu">
+    <ul class="list1_menu">
+        <li class="list1">HOTELS</li>
```
Comment (Lanng21) [inline src/components/menu.html:9]: i think u should move to component and use list tag to list item
example: [list](https://github.com/baonguyen26/html-css/blob/main/src/components/list.html)

File: src/components/menu.html [hunk 6]
```diff
@@ -1 +1,23 @@
...
+        <li class="list1">AIRLINES</li>
+        <li class="list1">VACATIONS</li>
+        <li class="list1">FIND MORE</li>
+    </ul>
+</div>
+
+<div class="wrap_list2_menu">
+    <ul class="list2_menu">
```
Comment (Lanng21) [inline src/components/menu.html:17]: same

File: src/components/menu.html [hunk 7]
```diff
@@ -1 +1,23 @@
...
+        <li class="lits2"><img class="img_lits2" src="../images/twitter_white.png" alt=""></li>
+        <li class="lits2"><img class="img_lits2" src="../images/facebook_white.png" alt=""></li>
+        <li class="lits2"><img class="img_lits2" src="../images/instagram_white.png" alt=""></li>
+    </ul>
+</div>
+</div>
```
Comment (Lanng21) [inline src/components/menu.html:23]: end of line

File: src/stylesheet/Main.scss [hunk 8]
```diff
@@ -1 +1,110 @@
...
+body{
+    font-family: Roboto;
+}
+.wrap_header{
+    display: block;
+}
+.image_header{
+    z-index: -1;
+    width: 100%;
+    height: 768px;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss:16]: don't fixed width height

File: src/stylesheet/Main.scss [hunk 9]
```diff
@@ -1 +1,110 @@
...
+    padding: 3%;
+    justify-content: space-between;
+    align-items: center;
+}
+.wrap_trademark{
+    display: flex;
+    justify-content: center;
+}
+.trademark{
+    color: #EBCB80;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss:35]: move to variable

File: src/stylesheet/Main.scss [hunk 10]
```diff
@@ -1 +1,110 @@
...
+}
+.list1_menu{
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    list-style: none;
+    column-gap: 30px;
+}
+.list1{
+    color: #FFF;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss:55]: same

File: src/stylesheet/Main.scss [hunk 11]
```diff
@@ -1 +1,110 @@
...
+}
+.list2_menu{
+    display: flex;
+    list-style: none;
+    justify-content: space-evenly;
+    align-items: center;
+    column-gap: 30px;
+}
+.img_lits2{
+    width: 25px;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss:74]: dont fixed w h for img

File: src/stylesheet/Main.scss [hunk 12]
```diff
@@ -1 +1,110 @@
...
+}
+.wrap_components{
+    display: block;
+    text-align: center;
+}
+.compo1{
+    margin-bottom: 30px;
+}
+.head_header{
+    color: #FFF;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss:90]: same

File: src/stylesheet/Main.scss [hunk 13]
```diff
@@ -1 +1,110 @@
...
+    text-align: center;
+    font-family: Roboto;
+    font-size: 44px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 48px; /* 109.091% */
+    letter-spacing: -1.6px;
+    width: 550px;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss:98]: same

File: src/stylesheet/Main.scss [hunk 14]
```diff
@@ -1 +1,110 @@
...
+}
+.wrap_video_header{
+    display: flex;
+    justify-content: center;
+    margin-bottom: 100px;
+}
+.video_header{
+    border: transparent 0px solid;
+    width: 555px;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss:107]: same

File: src/stylesheet/Main.scss [hunk 15]
```diff
@@ -1 +1,110 @@
...
+    margin-bottom: 30px;
+}
+.head_header{
+    color: #FFF;
+    text-align: center;
+    font-family: Roboto;
+    font-size: 44px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 48px; /* 109.091% */
```
Comment (Lanng21) [inline src/stylesheet/Main.scss:96]: delete cmt

**Review by Lanng21 — COMMENTED** [review]
I see you use scss but are you styling it the same way as css?
pls check git convention

File: src/components/menu.html (hunk 5 above)
Comment (Quannichan) [reply src/components/menu.html:9]: i will fix that on next pull request

File: src/components/header_compo.html (hunk 3 above)
Comment (Quannichan) [reply src/components/header_compo.html:10]: i will fix that!

File: src/stylesheet/_backgrounds.scss (hunk 1 above)
Comment (Quannichan) [reply src/stylesheet/_backgrounds.scss:1]: i was fix that!

**Pull request closed without merging**